src/lib/content/coordinators/GetContentItemV2Impl.ts

```diff
@@ -53,8 +54,16 @@ export class GetContentItemV2Impl
     try {
       response = await this.contentClient.get(url);
     } catch (err) {
-      if (err.response && err.response.status === 404) {
-        throw new Error(`Content item "${value}" was not found`);
+      if (err.response) {
+        if (err.response.status === 404) {
+          throw new HttpError(
```

Comment:
We should be throwing the same error for both v1 & v2 implementations.

Reply:
Noted - this was one of the things I wasn't sure about, since part of the request was to handle http errors, and technically this is one.

Reply:
Done.